src/helpers/BasicPointsAndBadges/PointsAndBadges.ts

```diff
@@ -38,4 +53,13 @@ async function awardPointsForActivity(activity: BasicPointsForBadgesActivity, ac
         case "custom":
             return await awardCustomActivityPoints(activity as CustomActivity, participantInfo);
     }
+}
+
+export function sumActivityPoints(activityStates: { [key: string]: BasicPointsForBadgesActivityState }) {
+    let pointsAccrued = 0;
+    for (const [key, value] of Object.entries(activityStates)) {
```

Comment:
Minor:  Possible rename suggestion.

`const [_, activityState]`

The `_` just means an ignored variable.